### ISSUE TITLE
Repeated prints of "Using external babel configuration"

### ISSUE THREAD

**Issue by sin-ack** [vercel/next.js#28499]
### What version of Next.js are you using?

11.1.0

### What version of Node.js are you using?

16.3.0

### What browser are you using?

Firefox

### What operating system are you using?

Gentoo GNU/Linux

### How are you deploying your application?

next start

### Describe the Bug

When using `next dev` to run my application in development mode with Next.js 11.1, the console gets repeated "Using external babel configuration" lines. This used to only be a single line before Next.js 11.1.

The issue seems to have started after #26570.

### Expected Behavior

For the message to only print once.

### To Reproduce

1. Have an external babel configuration at `.babelrc` 
2. Start an application with `next dev`
3. See lines printed

**Comment by ijjk** [comment]
Hi, this appears to be a duplicate of https://github.com/vercel/next.js/issues/27982 which is fixed in the latest version of Next.js `v11.1.1-canary.14`

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
